tests/integration/SdkTest_test.cpp

```diff
@@ -4585,9 +4586,6 @@ TEST_F(SdkTest, SdkSimpleCommands)
     logout(0);
     err = synchronousGetMiscFlags(0);
     ASSERT_EQ(MegaError::API_OK, err) << "Get misc flags failed (error: " << err << ")";
-
```

Comment:
Yes, that's right - the test was still problematic after that last solution though, because if the test fails before the re-login, that is not performed.   This time I'm marking the saved session as invalid so that the next test will perform the full login if needed.  thanks